content/gotraining.slide

```diff
@@ -92,23 +95,28 @@ Go favours simplicity and directness leading to some purposeful omissions:
 - No macros
 - No arguments about code style :)
 
+
 * Motivation and Demo
 
+
 * Why Go at ANZ
 
 - simple: easy to learn and read
 - a single obvious right way to do most things
-- tools - gofmt, goimports, go mod, go test, golangci-lint, coverage
+- tools - `gofmt`, `goimports`, `go` `mod`, `go` `test`, `golangci-lint`, coverage
```

Comment:
Should `go` `mod` be `go mod`? Same for `go` `test`.

Reply:
Could also allude to more tools available. Eg: CPU and memory profiling, race condition checking. Maybe best not to write everything these down but good notes to have on you.

Reply:
yup, we had this discussion on the channel. This is a bug/feature of go present - it only works with single words, but the way I've done it renders the desired result.

Reply:
Fair enough 👍